packages/react-components/react-combobox/src/components/Combobox/Combobox.test.tsx

```diff
@@ -420,4 +420,137 @@ describe('Combobox', () => {
     fireEvent.keyDown(combobox, { key: 'ArrowUp' });
     expect(combobox.getAttribute('aria-activedescendant')).toEqual(getByText('Green').id);
   });
+
+  it('should move active option based on typing', () => {
+    const { getByTestId, getByText } = render(
+      <Combobox open data-testid="combobox">
+        <Option>Red</Option>
+        <Option>Green</Option>
+        <Option>Blue</Option>
+      </Combobox>,
+    );
+
+    const combobox = getByTestId('combobox');
+
+    fireEvent.change(combobox, { target: { value: 'g' } });
```

Comment:
_Note:_ For components like Combobox I suggest to use [`user-event`](https://testing-library.com/docs/user-event/intro) as it simulates full interactions.

Reply:
oh, good call 👍

Reply:
good call 👍